crates/serpen/tests/fixtures/test_edge_cases.py

```
@@ -0,0 +1,26 @@
+from ..parent import helper
+from ...grandparent.deep import nested_module
+from .relative_module import *
+import importlib
+from importlib import import_module
+from concurrent.futures import ThreadPoolExecutor, ProcessPoolExecutor
+
+
+# Some actual code to make it valid
+def main():
+    data = defaultdict(list)
+    path = PathAlias("/tmp")
```

Ruff (0.11.9)

Local variable `path` is assigned to but never used

Remove assignment to unused variable `path`

(F841)